src/Microsoft.AspNetCore.Mvc.Core/AuthorizeFilter.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
I'm fine with a .Authorization namespace, though we need to be careful about what we move there. I want to slow down significantly on moving stuff, but if there's a good set of stuff that is likely more obscure anyway, let's rip the Band-Aid off and do it.